src/Symfony/Component/Notifier/Bridge/Yunpian/YunpianTransportFactory.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Yunpian;
+
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\AbstractTransportFactory;
+use Symfony\Component\Notifier\Transport\Dsn;
+use Symfony\Component\Notifier\Transport\TransportInterface;
+
+/**
+ * @author Mathieu Santostefano <msantostefano@protonmail.com>
+ */
+class YunpianTransportFactory extends AbstractTransportFactory
+{
+    /**
+     * @return YunpianTransport
+     */
+    public function create(Dsn $dsn): TransportInterface
+    {
+        if ('yunpian' !== $dsn->getScheme()) {
+            throw new UnsupportedSchemeException($dsn, 'yunpian', $this->getSupportedSchemes());
+        }
+
+        $apiKey = $this->getUser($dsn);
+
+        return (new YunpianTransport($apiKey, $this->client, $this->dispatcher))
```

Comment:
One line please

Reply:
fixed. I put the host value in a variable, to avoid a too long line.